extensions/api/data-management/asset/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/model/DataAddress.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Diego Gomez
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *   Diego Gomez - Initial API and Implementation
+ */
+
+package org.eclipse.dataspaceconnector.api.datamanagement.asset.model;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = DataAddress.Builder.class)
+public class DataAddress {
```

Comment:
Please rename to `*Dto`. Sorry, overlooked this in the issue spec, which I'll update.